src/Report/Text.php

```diff
@@ -69,7 +69,7 @@ public function prepareReportForMutant(Result $result)
     }
     
     /**
-     * 
+     *
```

Comment:
StyleCI can do this automatically if you enable more fixers than just those in the psr2 preset.